modules/administration_manual/pages/release_notes.adoc

```diff
@@ -17,6 +18,65 @@
+
+=== Federation: Compliance with proposed OpenCloudMesh 1.0 specification
+
+Federation enables instances of ownCloud and other supporting platforms to exchange information. It allows users to share data across installations building a worldwide collaboration network of decentralized nodes - each under the full control of it's provider. Together with the other vendors the underlying OpenCloudMesh API specification has been shifted to a new level to clean up the interface, improve it's stability and to set the foundation for future feature improvements. ownCloud Server 10.1 is compliant with the new specification proposal. The introduction of the new specification does not involve changes in functionality for users.
+
+=== New Collaborative Tags Scope: Static Tags
+
+Version 10.1 comes with a new scope for Collaborative Tags called "Static Tags". In addition to the https://doc.owncloud.com/server/10.0/user_manual/files/webgui/tagging.html#filter-by-tag[other tag scopes], these tags are intended to be supplied by administrators and linked with policies in the https://marketplace.owncloud.com/apps/firewall[File Firewall], https://marketplace.owncloud.com/apps/files_classifier[Document Classification] or https://marketplace.owncloud.com/apps/workflow[Workflows], for example. Every user will be able to see these tags assigned to files but only users in specified groups have the permission to assign or unassign them. This makes it possible to equip certain users with the means to impose pre-defined policies upon files. To create such tags administrators need to use the https://marketplace.owncloud.com/apps/systemtags_management[Collaborative Tags Management] extension.
+
+=== Other notable changes
+
+- The user/group deletion in the users page now has a confirmation dialog to prevent unintentional user deletion
+- The default public link share name has been changed to be "Public link" instead of formerly the file or folder's name
+- **Allow loading JSON files in setups with pretty URLs**
+**=> Please check .htaccess to make sure this**
+
+
+=== Solved known issues
+
+- LDAP users can upload avatars again https://github.com/owncloud/core/issues/33369[#33369]
+- Versions list performance improvements https://github.com/owncloud/core/issues/33291[#33291]
+- Improved compatibility with 3rd party WebDAV applications (fixed PROPFIND with depth infinity requests through Sabre update) https://github.com/owncloud/core/issues/28341[#28341]
+- Fixed `occ encrypt-all` command to not attempt re-encrypting already encrypted files https://github.com/owncloud/core/issues/33206[#33206]
+
+=== Known issues
+
+Currently there are no known issues with ownCloud Server 10.1.
```

Comment:
we have some related to persistent locking but they'll likely disappear until final
I'll let you know if anything remains